### DISCUSSION TITLE
Slow Boot Time and Timeout on Fresh Wave 3.0 Install

### DISCUSSION THREAD

**Discussion by webtechlabs** [thedevdojo/wave#146] [Q&A] [answered]
On a fresh Wave 3.0 install.  Empty database / empty install folder.  Following every step in the manual installation instruction.  I'm typically getting 11+ second request durations as identified by Google Inspect as well as Laravel Debugbar.  Fresh install of Filament and even Wave 2.0 install I was fooling around with a week ago do NOT have same slow page refresh issue.  I've tried re-installing Wave a few times.  Nothing else is installed except Laravel Debugbar.

Home Page = Booting (11.11s), Application (1.39s)
Home Page = Booting (10.52s), Application (366ms)
Admin Page = Booting (10.48s), Application (486ms)
Admin Page = Booting (10.76s), Application (425ms)

Local Win 11 23H2 laptop with i7-13620H and 16GB RAM 
Nginx 1.27.2
PHP 8.3.12
MariaDB 11.5.2

I occasionally this error pops up:

```
Symfony \ Component \ ErrorHandler \ Error \ FatalError
PHP 8.3.12
Laravel 11.28.1
Maximum execution time of 30 seconds exceeded

D:\Temp\project\vendor\symfony\finder\Iterator\SortableIterator.php: 51

    private \Traversable $iterator;
    private \Closure|int $sort;

    /**
     * @param \Traversable<string, \SplFileInfo> $iterator
     * @param int|callable                       $sort     The sort type (SORT_BY_NAME, SORT_BY_TYPE, or a PHP callback)
     *
     * @throws \InvalidArgumentException
     */

    public function __construct(\Traversable $iterator, int|callable $sort, bool $reverseOrder = false)
    {
        $this->iterator = $iterator;
        $order = $reverseOrder ? -1 : 1;

        if (self::SORT_BY_NAME === $sort) {
            $this->sort = static fn (\SplFileInfo $a, \SplFileInfo $b) => $order * strcmp($a->getRealPath()

(2) LOGS

installed.json does not exist at: D:\Temp\gymrank2\resources\plugins/installed.json

Installed plugins: []
```

**Comment by bobbyiliev** [comment]
Thanks for reporting that! Adding a link to the discussion on the DevDojo forum where we are working on this:

> https://devdojo.com/question/wave-saas-v3-opens-pages-very-slow

**Reply by bobbyiliev** [reply, marked as answer]
The fix was to run the following:

```
php artisan filament:optimize
```

This caches the blade icons which were causing the issue.

Also if you are on Windows and have Windows Defender enabled, make sure to add an exclusion in Windows Defender for your whole web dev folder.
